nym-vpn-core/crates/nym-statistics-api-client/src/routes.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright 2025 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: GPL-3.0-only
+
+// pub(crate) const PUBLIC: &str = "public";
+pub(crate) const V1: &str = "v1";
```

Comment:
Andrew has been working towards using simple strings like `/v1/stats/report` instead of individual components for simplicity. Perhaps we should too?